### DISCUSSION TITLE
Error 503 Backend Fetch Failed While Creating New Customer Account

### DISCUSSION THREAD

**Discussion by TechDriven** [wardenenv/warden#709] [Help & Support]
Hi Team,

I have setup my magento project using warden and everything is working fine except while I am trying to create new user account I am getting this error **Error 503 Backend Fetch Failed**. I am new to warden and not sure how should I fix this?

It would be great if you suggest any solution to fix this issue?  

CC: @navarr 

![Warden_Post_Varnish_Issue](https://github.com/wardenenv/warden/assets/10269068/f5c13ab6-8f1c-4e05-9bcd-5bc957ee6449)

**Comment by navarr** [comment]
Backend fetch fails means that Nginx, or maybe even PHP is failing to return anything - it might be a timeout.

I'd double check your logs first and foremost

**Reply by TechDriven** [reply]
Hi @navarr 
Thank you for your comment. For now I have disable the Varnish and its working fine on my local development. For future similar issues, I will go through with nginx logs using `warden env logs --tail 0 -f nginx php-fpm php-debug`
